README.md

```diff
@@ -1,3 +1,4 @@
 # bulk-scan-orchestrator
 
 [![Build Status](https://travis-ci.org/hmcts/bulk-scan-orchestrator.svg?branch=master)](https://travis-ci.org/hmcts/bulk-scan-orchestrator)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/e9272daf4b714e4f95280916e763b6b2)](https://www.codacy.com/project/HMCTS/bulk-scan-orchestrator)
```

Comment:
link doesn't work.
I think it should be
https://www.codacy.com/app/HMCTS/bulk-scan-orchestrator ('app' not 'project')